app/assets/stylesheets/application.scss

```
@@ -43,6 +85,17 @@ $sidebar-width: 220px;
 /* self_registrations */
 @import 'self_registrations/select_resource';
 
+// Sizing for the inline SVG icons that replaced glyphicon-* (see
+// ApplicationHelper#icon). Scales with the surrounding font-size like the
+// glyphicon font did, and inherits color via currentColor.
+.icon-svg {
+  display: inline-block;
+  width: 1em;
+  height: 1em;
+  vertical-align: -0.15em;
+  fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)